processing/src/main/java/io/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -83,7 +87,169 @@ public byte asByte()
     }
   }
 
-  public static class Builder
+  @JsonCreator
+  public static DictionaryEncodedColumnPartSerde createDeserializer(
```

Comment:
now these methods in all impls should probably be called "createInstance" as "deserializer" is not what they return and that has different meaning after this PR.